<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubWindow</class>
 <widget class="QMainWindow" name="SubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SubWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="altonClient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="invalid">
     <widget class="QPushButton" name="invalidOpen">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QLabel" name="invalidTitel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[Test Application]

 Set the location of the sample file for testing.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="injectList">
     <widget class="QListWidget" name="fileList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>580</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="checkCancel">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkDelete">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="detailInject">
     <widget class="QLabel" name="detailTitle">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>460</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>EXNCP-40685</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="detailStart">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>90</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="detailBack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>90</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="detailContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>640</width>
        <height>540</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="detailSave">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="altonService" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>640</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 0);</string>
    </property>
    <widget class="QPushButton" name="altonServiceClear">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="altonServiceTitle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>460</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Alton Service</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="altonServiceContent" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>640</width>
       <height>530</height>
      </rect>
     </property>
     <widget class="QLabel" name="altonServiceTransaction1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>[Transaction]
SFC.High_Performance_Gauge.Constant.TurboGauge.Value 0.57
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>20</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="hmi" native="true">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>0</y>
      <width>640</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 0);
background-color: rgb(0, 170, 0);</string>
    </property>
    <widget class="QPushButton" name="hmiClear">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="hmiTitle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>460</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>HMI</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="hmiContent" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>640</width>
       <height>530</height>
      </rect>
     </property>
     <widget class="QLabel" name="hmiSubscription1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>[Subscription]
SFC.High_Performance_Gauge.Constant.TurboGauge.Value 0.57
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>20</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="companyInfo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>920</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuDelete">
     <property name="title">
      <string>Delete</string>
     </property>
     <addaction name="actionTAV_Delete"/>
     <addaction name="actionScript_Delete"/>
     <addaction name="actionInfo_Delete"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuDelete"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionVSM_Path"/>
    <addaction name="actionTAV_Path"/>
    <addaction name="actionAltonClient_Path"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionVSM_Path">
   <property name="text">
    <string>VSM Path</string>
   </property>
  </action>
  <action name="actionTAV_Path">
   <property name="text">
    <string>TAV Path</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpen_Dir">
   <property name="text">
    <string>Open Dir</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTAV">
   <property name="text">
    <string>TAV</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionAltonClient_Path">
   <property name="text">
    <string>Alton Client Path</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionTAV_Delete">
   <property name="text">
    <string>TAV</string>
   </property>
  </action>
  <action name="actionScript_Delete">
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionInfo_Delete">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
